package.json

```diff
@@ -1,11 +1,9 @@
 {
-  "name": "zk-state",
+  "name": "zk-states",
   "packageManager": "yarn@3.6.1",
   "private": true,
   "version": "0.0.1",
   "description": "Global React state management with ZK support",
```

Comment:
What about:
description: "Verifiable state manager for React based on SnarkyJS & Mina Protocol"